app/services/publisher_wallet_setter.rb

```diff
@@ -2,32 +2,28 @@
 class PublisherWalletSetter < BaseApiClient
```

Comment:
Minor: This job doesn't directly set the wallet anymore, so maybe we should add a comment up top clarifying that it sets uphold params.